__tests__/news/news.query.test.js

```diff
@@ -1,14 +1,19 @@
 /* eslint-disable no-undef */
 const { gql } = require('apollo-boost');
 const client = require('../../utils/apollo-test-client');
-require('dotenv').config();
+const loginAdminOperation = require('../../utils/loginAdmin');
 const { NEWS_NOT_FOUND } = require('../../error-messages/news.messages');
 
+require('dotenv').config();
+
 let newsId = '';
+let token;
 const newsDoesNotExistId = '5f311ec5f2983e390432a8c3';
 
 describe('querries', () => {
   beforeAll(async () => {
```

Comment:
Please, add here two `describe` statements, one for all tests related to the bulk get, another - for the get by id. Also add tests that both logged-in users, users without token and admins can properly perform bulk get and get by id operations